DOCS.md

```diff
@@ -28,7 +27,7 @@ The following is a sample Docker configuration in your .drone.yml file:
 publish:
   docker:
     username: kevinbacon
-    password: pa55word
+    password: $$DOCKER_PASSWORD
```

Comment:
Should there maybe be a section offering suggestions on what should be secured? The $$ is best practices but might also make things less clear.